lib/erb_lint/cli.rb

```diff
@@ -35,7 +35,11 @@ def run(args = ARGV)
       load_config
 
       if !@files.empty? && lint_files.empty?
-        failure!("no files found...\n")
+        if allow_no_files?
+          success!("no files found...\n")
+        else
+          failure!("no files found...\n")
+        end
       elsif lint_files.empty?
         failure!("no files found or given, specify files or config...\n#{option_parser}")
```

Comment:
Should we use option to determine exit status here as well? It feels like this should still be an error.

Reply:
I think this should be an error.